rocAL/rocAL/include/master_graph.h

```diff
@@ -124,8 +136,9 @@ class MasterGraph
     RaliAffinity _affinity;
     const int _gpu_id;//!< Defines the device id used for processing
     pLoaderModule _loader_module; //!< Keeps the loader module used to feed the input the images of the graph
+    pVideoLoaderModule _video_loader_module; //!< Keeps the video loader module used to feed the input sequences of the graph
     TimingDBG _convert_time;
-    const size_t _user_batch_size;//!< Batch size provided by the user
```

Comment:
why this is changed to non const parameter?

Reply:
The _user_batch_size will be initially set to user given batch_size value.
For the video readers we would be processing batch_size * sequence_length number of frames.
Hence we would need to modify the _user_batch_size parameter to user given batch_size * sequence_length.